app/models/medium.rb

```diff
@@ -102,6 +105,10 @@ def put_file_with_retry(file_path, opts={}, num_retries=5)
     raise e
   end
 
+  def validate_content_type
+    errors.add(:content_type, 'Invalid file type') unless ALLOWED_CONTENT_TYPES.match?(content_type)
```

Comment:
any reason this regex doesn't match the whole content type? 

i think i could smuggle unallowed media resources in if they contain an allowed substring, e.g. `text/html-json` would pass this test and allow the media resource to be saved...it may fail to set an actual path on the validation via https://github.com/zooniverse/panoptes/blob/85142aa1bc5c57f8ec0164174aa7583a885bcbb9/app/models/medium.rb#L35 though..

still this should be more robust.